CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # nf-core/tools: Changelog
 
+## [v2.7.2 - Mercury Eagle Patch](https://github.com/nf-core/tools/releases/tag/2.7.2) - [2022-12-19]
+
+### Template
+
+- Fix the syntax of github_output in GitHub actions ([#2114](https://github.com/nf-core/tools/pull/2114))
+- Fix a bug introduced in 2.7 that made pipelines hang ([#2132](https://github.com/nf-core/tools/issues/2132))
+
+### Linting
+
+- Allow specifying containers in less than three lines ([#2121](https://github.com/nf-core/tools/pull/2121))
+- Run prettier after dumping a json schema file ([#2124](https://github.com/nf-core/tools/pull/2124))
```

Comment:
I would put this in "General" instead of "Linting" or is the dumping part done during a linting step?

Reply:
Not dumping but prettier

Reply:
yes, but here we use it to format a file not really to lint it. But your logic also makes sense. 🤔